include/NumericAddressedBackend.h

```diff
@@ -158,17 +158,17 @@ namespace ChimeraTK {
     /// mutex for protecting unaligned access
     std::mutex _unalignedAccess;
 
-    template<typename UserType>
-    boost::shared_ptr<NDRegisterAccessor<UserType>> getRegisterAccessor_impl(
-        const RegisterPath& registerPathName, size_t numberOfWords, size_t wordOffsetInRegister, AccessModeFlags flags);
-
     friend NumericAddressedLowLevelTransferElement;
     friend TriggeredPollDistributor;
 
     template<class UserType, class ConverterType>
     friend class NumericAddressedBackendMuxedRegisterAccessor;
 
    private:
+    template<typename UserType>
```

Comment:
Wow, yes. Is is now private so the derived class cannot call it and has to use the correct vtable override. I like it!